src/vs/platform/instantiation/test/common/instantiationService.test.ts

```diff
@@ -382,7 +382,7 @@ suite('Instantiation Service', () => {
 		// and then Servce2 should not be created a second time
 		const Service21 = createDecorator<any>('service21');
 		class Service21Impl {
-			constructor(@Service2 readonly service2: Service2Impl, @Service1 readonly service1: Service1Impl) { }
+			constructor(@Service2 public readonly service2: Service2Impl, @Service1 public readonly service1: Service1Impl) { }
```

Comment:
do you need the `public`s?

Reply:
In this case we do, since the tests check this property